### ISSUE TITLE
Date interpreted TZ incorrectly

### ISSUE THREAD

**Issue by da11an** [gaborcsardi/parsedate#26]
> parsedate::parse_date("2010-07-01")
[1] "2010-06-30 19:00:00 CDT"

CDT is UTC -5, so I think that's why it bumped my time 5 hours forward. But wouldn't it better to assume that the data is for the same timezone as the computer viewing it, unless otherwise specified?

**Comment by gaborcsardi** [comment]
Yes, dates without an explicit time zone are assumed to be UTC. Whether that's a good choice is debatable, but it is not something I can change at this point. Maybe with a `parse_local_date()` function?

**Comment by da11an** [comment]
parse_local_date() would do it!
